doc/xml/reference.xml

```diff
@@ -210,6 +210,24 @@
                         <example>120</example>
                     </config-key>
 
+                    <!-- ======================================================================================================= -->
+                    <config-key id="job-retry" name="Job Retry Count">
+                        <summary>Retry count for local jobs.</summary>
+
+                        <text>The number of retries for jobs run in the local process, e.g. file backup.</text>
+
+                        <example>5</example>
+                    </config-key>
+
+                    <!-- ======================================================================================================= -->
+                    <config-key id="job-retry-interval" name="Job Retry Interval">
+                        <summary>Retry interval after first retry.</summary>
+
+                        <text>The first retry is always performed immediately. This options specifies the interval to wait (in seconds) before performing each additional retry.</text>
```

Comment:
`This options` should not be plural: `This option`

Reply:
Fixed in commit 2023430cbc97bb5deaf0f28372ecaebc1bbce3f7

Reply:
Looks good to me.